docs/development_guidelines.md

```diff
@@ -448,6 +448,65 @@ func foo(a, b, c,
 }
 ```
 
+### Inline slice definitions
+
+In Go a list of slices can be initialized with values directly, using curly
+braces. Whenever possible, the more verbose/indented style should be used for
```

Comment:
tho one is parentheses and the other is curly braces, it feels like this somewhat contradicts with the reasoning here,

> As long as the visual symmetry of the opening and closing parentheses is
preserved, arguments that would otherwise introduce a new level of indentation
are allowed to be written in a more compact form.

Reply:
yeah it's more about the preference here - like do we prefer compactness over verboseness, or is it case-specific?

Reply:
I actually prefer the more compact version, but Laolu prefers the more verbose one (probably because it is easier to insert another element with copy/paste and also makes git conflicts a bit easier to handle). So that's the reason I wrote it this way.

Reply:
Okay, I looked at this again. And I think we don't necessarily contradict ourselves here. The part about the visual symmetry for wrapping long function calls is about what line you place the opening vs. the closing curly braces of a slice.
This section (about inline slice definitions) is about how many indentation levels you create when you directly initialize a slice with values (vs. just initializing simple structs in function/method calls).

So to align with Laolu's preferences, I think we should prefer the more-indented version but allow the compact version when things get too long.

Does that work for you or would you invert that (prefer compact but allow more verbose, but then with what rationale)?